src/components/FeatureAttributes/fields/FeatureAttributeLocaleField/FeatureAttributeLocaleField.tsx

```diff
@@ -27,14 +27,15 @@ export const FeatureAttributeLocaleField: FC<
   FeatureAttributeLocaleFieldProps
 > = ({ dataType, ...props }) => {
   const { t } = useTranslation(i18n.namespace);
-  const { fieldTextProps } = useContext(FeaturesAttributesContext);
+  const { fieldTextProps, purposes } = useContext(FeaturesAttributesContext);
   const form = useFormContext();
-  const required = false;
 
-  if (dataType !== "formatted_date_time") {
+  if (dataType !== "formatted_date_time" || !purposes.includes("synthesis")) {
```

Comment:
This is not specific to synthesis, if you have a datetime you can specify the locale

Reply:
(What does the sytem use feature attribute locale for?)[https://teams.microsoft.com/l/message/19:41a21c3cf80f4764b3b8b0c6a076b022@thread.skype/1727097976599?tenantId=479f98a7-1134-46ca-b954-068006331327&groupId=15ead0bb-6ff0-4d3d-82a2-d9901d1d7fa2&parentMessageId=1727097976599&teamName=Engineering&channelName=Recipes%2C%20Demos%2C%20Docs%2C%20Onboarding&createdTime=1727097976599].

Reply:
Marked Locale as core